torch/_inductor/codegen/mps.py

```diff
@@ -79,6 +79,11 @@ def index_expr(expr: sympy.Expr, dtype: torch.dtype) -> str:
         )
         return ops.to_dtype(var, dtype)
 
+    @staticmethod
+    def masked(mask: CSEVariable, body: sympy.Expr, other: CSEVariable) -> str:
+        # TODO: Add a proper implementation considering there are no lambdas in Metal
```

Comment:
We don't actually codegen lambdas here.  For triton/halide this gets codegened to adding masking to any loads inside the body() function.  On CPU it becomes an `if`.